src/hotspot/share/utilities/filterQueue.inline.hpp

```diff
@@ -112,4 +112,29 @@ E FilterQueue<E>::pop(MATCH_FUNC& match_func) {
   } while (true);
 }
 
```

Comment:
Need to update the copyright for this file.

Reply:
Fixed